napari/layers/base/base.py

```diff
@@ -11,6 +11,7 @@
 import numpy as np
 
 from ..._vendor.cpython.functools import cached_property
+from ...components.dims import reorder_after_dim_reduction
```

Comment:
this import needs to be delayed (tuck it into `_update_dims`) ... that's why at least one of the tests is failing

Reply:
right, thanks! I should not have been lazy and just run layers and components tests locally!